src/main/java/dev/chuckbot/controller/JokeController.java

```diff
@@ -24,6 +25,8 @@ public JokeController(ChuckNorrisJokeService service) {
         this.service = service;
     }
 
+    private int sortCount = 0;
```

Comment:
This cannot work for different users as the "state" of the sort is thread-specific, see diagram.
![image](https://user-images.githubusercontent.com/2828581/142585284-05f094cf-7db2-4565-8a59-f21f71e761bd.png)